src/client/datascience/commands/commandRegistry.ts

```diff
@@ -552,3 +562,26 @@ export class CommandRegistry implements IDisposable {
         }
     }
 }
+
+export function getImportCodeForFileType(filepath: string) {
+    const fileExtension = path.extname(filepath);
+    let code = 'import pandas as pd\n';
+    switch (fileExtension) {
+        case '.csv':
+            code += `df = pd.read_csv(r"${filepath}")`;
+            break;
+        case '.xlsx': // TODO dependency check for openpyxl
```

Comment:
TODOs are ok, but are they being tracked somewhere? They always make me nervous going in, easy to forget. If you have more might want to tag them for easy searching. Like (DataWranglerTODO).

Reply:
Good point, removed them